src/quo2/components/messages/system_message.cljs

```diff
@@ -31,15 +31,6 @@
           ((theme/get-theme) themes)
```

Comment:
it would be great if we could update this component to current best practices while here, e.g use new theming approach, style and view namespaces etc 👍

Reply:
do we have documentation how to that ?